src/module/party/party-xp.js

```diff
@@ -27,38 +32,42 @@ export class OsePartyXP extends FormApplication {
 
   /**
    * Construct and return the data object used to render the HTML template for this form application.
-   * @return {Object}
+   *
+   * @returns {object}
    */
   getData() {
-    let data = {
+    return {
       actors: OseParty.currentParty,
       data: this.object,
       config: CONFIG.OSE,
       user: game.user,
-      settings: settings,
+      settings: game.settings,
     };
-    return data;
   }
 
-  _onDrop(event) {
+  // eslint-disable-next-line no-underscore-dangle
+  _onDrop(event) { // eslint-disable-line class-methods-use-this
     event.preventDefault();
     // WIP Drop Item Quantity
     let data;
     try {
       data = JSON.parse(event.dataTransfer.getData("text/plain"));
       if (data.type !== "Item") return;
-    } catch (err) {
+    } catch (error) {
       return false;
     }
   }
   /* -------------------------------------------- */
 
-  _updateObject(event, formData) {
+  // eslint-disable-next-line no-underscore-dangle
+  _updateObject(event) {
+    // eslint-disable-next-line no-underscore-dangle
     this._dealXP(event);
   }
 
-  _calculateShare(ev) {
-    const currentParty = OseParty.currentParty;
+  // eslint-disable-next-line no-underscore-dangle
+  _calculateShare() {
```

Comment:
use # here?

Reply:
We should be able to use # to mark private class members anywhere that Foundry doesn't expect a dangling underscore. I'd like to do this in a separate PR, since there are so many instances of things like this to change.